Dockerfile

```
@@ -1,5 +1,7 @@
 FROM clojure:openjdk-14-tools-deps
 
+RUN apt-get update && apt-get install -y git
```

Could you change as below?
c.f. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

```suggestion
RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
```